docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,363 @@
+4. [Configure the UID2 mobile SDK](#configure-the-uid2-mobile-sdk).
+5. [Check that the token was successfully generated and then pass it for bid stream use](#pass-generated-token-for-bid-stream-use).
+6. [Optionally, integrate the UID2 GMA/IMA Plugin for GAM Secure Signal integration](#optional-uid2-gmaima-plugin-for-gam-secure-signal-integration).
+
+<!-- It includes the following sections:
+
+- [Mobile SDK Version](#mobile-sdk-version)
+- [Overview](#overview)
+- [Complete the UID2 Account Setup](#complete-the-uid2-account-setup)
+- [Client-Server Mobile Integration Data Flow Overview](#client-server-mobile-integration-data-flow-overview)
+- [Implement Server-Side Token Generation](#implement-server-side-token-generation)
+- [Server-Side Token Refresh](#server-side-token-refresh)
+- [Add the UID2 Mobile SDK into Your Mobile App](#add-the-uid2-mobile-sdk-into-your-mobile-app)
+- [Using the UID2 Integration Environment](#using-the-uid2-integration-environment)
+- [Optional: Reduce Latency by Setting the API Base URL for the Production Environment](#optional-reduce-latency-by-setting-the-api-base-url-for-the-production-environment)
+- [Token Storage](#token-storage)
+- [Pass Generated Token for Bid Stream Use](#pass-generated-token-for-bid-stream-use)
+- [When to Pass DII into the SDK](#when-to-pass-dii-into-the-sdk)
+- [Enable Logging)](#enable-logging)
+- [Enable Automatic Token Refresh in Mobile App/Client Side](#enable-automatic-token-refresh-in-mobile-appclient-side)
+- [Optional: UID2 GMA/IMA Plugin for GAM Secure Signal integration](#optional-uid2-gmaima-plugin-for-gam-secure-signal-integration) -->
+
+
+## Mobile SDK Version
+
+This guide provides instructions for using version v1.2.0 or higher of either of these UID2 mobile SDKs:
+
+- UID2 SDK for Android
+- UID2 SDK for iOS
+
+For instructions for installing the correct SDK/version into your mobile app, see [Add the UID2 Mobile SDK into Your Mobile App](#add-the-uid2-mobile-sdk-into-your-mobile-app).
+
+
+## Complete the UID2 Account Setup
+
+To set up your account, follow the steps described in [Account Setup](../getting-started/gs-account-setup.md).
+
+When account setup is complete, you'll receive your unique API key and client secret. These values are unique to you, and it is important to keep them secure. For details, see [API Key and Client Secret](../getting-started/gs-credentials.md#api-key-and-client-secret).
+
+## Client-Server Mobile Integration Data Flow Overview
+
+The following diagram shows the data flow that the publisher must implement for UID2 client-server mobile integration.
+
+This example uses the [UID2 SDK for Android](../sdks/uid2-sdk-ref-android.md) in the client-side mobile app and the [UID2 SDK for Java](../sdks/uid2-sdk-ref-java.md) on the server side.
+
+**GWH__KL_01: diagram updates to remove TTD colors at least?**)
+
+![Mobile Client-Server Integration Example](images/integration-mobile-client-server.png)
```

Comment:
Probably not needed for this PR but would be good to include a zoom tool so we can click an image and it will be enlarged for easier reading. It seems we can only do this with a docusaurus plugin.

https://gabrielcsapo.github.io/docusaurus-plugin-image-zoom/docs/getting-started/

Meanwhile, should we state something like "Open image in new tab to see the enlarged diagram"?